Gordon360/Services/HousingService.cs

```diff
@@ -34,36 +34,73 @@ public IEnumerable<HousingViewModel> GetAll()
 
         /// <summary>
         /// Saves student housing info
+        /// - first, it creates a new row in the applications table and puts the id of the primary applicant and a timestamp
```

Comment:
In "puts the id of the ...", it would be more appropriate and less confusing to use "insert" instead of "put"

Reply:
I agree, and saying "... and inserts the id of the ..." makes it more clear grammatically as well